Code/Framework/AzManipulatorTestFramework/Include/AzManipulatorTestFramework/ImmediateModeActionDispatcher.h

```diff
@@ -106,4 +113,13 @@ namespace AzManipulatorTestFramework
     {
         return GetMouseInteractionEvent()->m_mouseInteraction.m_keyboardModifiers;
     }
+
+    inline AZStd::chrono::milliseconds ImmediateModeActionDispatcher::EditorViewportInputTimeNow()
```

Comment:
Q: is this something that should be specific to the immediate mode action dispatcher or something more general for the  parent action dispatcher class? this might require a broader conversation about the class design once the retained mode action dispatcher is removed but just want to be sure we're not bleeding generic functionality into this specialized class when it might make more sense to live in the parent class.

Reply:
That's a good question, right now I don't think it's a huge problem as there is only the `ImmediateModeActionDispatcher` - it might be worth us refactoring all the code to just `ActionDispatcher` in future as Retained has now been removed, this could be followed-up with in a subsequent PR